.github/workflows/claude.yml

```
@@ -39,26 +36,28 @@ jobs:
+            Follow our coding standards
+            Ensure all new code has tests
+            Use TypeScript for new files
+            Always respond in Japanese (日本語で回答してください)
+            PR ページの肥大化を防ぎタイムラインの全容を掴みやすくするために、Claude が作成する GitHub コメントは、コメント全体/Markdown 全要素を "<details>" タグによる一つの折りたたみパネルの中に収める
+            作業上必要なツールは OS にインストールして良い
+
           # Optional: Custom environment variables for Claude
           # claude_env: |
-          #   NODE_ENV: test
-
+          #   NODE_ENV: test
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)